config/converge.go

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"sort"
+
+	configpb "github.com/GoogleCloudPlatform/testgrid/pb/config"
+	"github.com/golang/protobuf/proto"
+)
+
+// Zero-memory member for hash sets
+type void struct{}
+
+var insert void
```

Comment:
I don't know what word in the statement `someSet[thingToInsert] = ****` does convey that well. Ideas would be appreciated, because I'm probably going to need to use a set again in golang.

Reply:
"inserted"??